### ISSUE TITLE
Hey that's cool and all and I'm gonna let you finish but first 

### ISSUE THREAD

**Issue by m1ndflay3r** [cynicade/AoC2015#1]
I did everything wrong but vending machine is still broken. plz fix

![vending-machine-broken-hed2-2014](https://user-images.githubusercontent.com/47760937/94361457-a5544580-0071-11eb-8531-af2b917a6f2e.jpg)

**Comment by m1ndflay3r** [comment]
0/10 doesn't run on commodore 64

**Comment by m1ndflay3r** [comment]
Will this let me run Fortnite??? at 61729P???

**Comment by m1ndflay3r** [comment]
Instructions unclear, got stuck in vending machine

**Comment by m1ndflay3r** [comment]
This is pretty cool but can you rewrite it in brainfuck kthx

**Comment by m1ndflay3r** [comment]
0/10 can't fit on a single punchcard

**Comment by m1ndflay3r** [comment]
Ran several times but Area of Ceffect never triggered 0/10 terrible for boss fights

**Comment by m1ndflay3r** [comment]
There's no clear way to upload this to a TI-83???????????????????????

**Comment by m1ndflay3r** [comment]
ran this exactly once and my dad left to go get cigarettes shortly after. rating 2/10 because he never came back

**Comment by m1ndflay3r** [comment]
Does this support HHTTPPSS?

**Comment by m1ndflay3r** [comment]
Managed to install on TI-83 but it was a pretty bad doom clone so I deleted it 0/10

**Comment by m1ndflay3r** [comment]
Ran this once and immediately got a notification that my house was targeted from orbit. Haven't heard anything since but I'm scared. Plz fix

**Comment by m1ndflay3r** [comment]
Had this printed in book form, but it doesn't make for very good reading. 0/10 hard to follow plot

**Comment by m1ndflay3r** [comment]
Vending machine is still broken??? why isn't this fixed????

**Comment by m1ndflay3r** [comment]
So I uploaded this to my alexa and it immediately broke up with me and took half my stuff in the fallout. 0/10

**Comment by m1ndflay3r** [comment]
Does this run on Sega Dreamcast? Asking for a friend

**Comment by m1ndflay3r** [comment]
Uploaded this to my Neopet and now it's addressing me by my first name and insists I feed it meat. Tried taking the battery out but that just made it hungrier. Plz fix

**Comment by m1ndflay3r** [comment]
Update: My dad came back but only long enough to steal my wallet and call me a bitch. changing rating from 2/10 to 0/10

**Comment by cynicade** [comment]
Yeah, all those are features. I just forgot to update the README. Thank you for beta testing though!

**Comment by m1ndflay3r** [comment]
Pfft, should still be in alpha imo. I ran this exactly *once* and now I'm a raging homosexual. Don't want to find out what happens if I run it again as I literally can't be any gayer. 0/10

**Comment by cynicade** [comment]
I ran this a few times during development. I'm now not allowed within 2km of a zoo. Proceed with caution.

**Comment by m1ndflay3r** [comment]
Ran this yesterday and it started complaining about running out of memory. I asked it what memory it needed and it said it needed mine. I thought I could appease it by giving it just a little, but it just keeps taking more of my memories. Went to my grandmother's birthday shortly thereafter and realised I didn't recognise her face but I just rolled with it. Took me 3 hours to realise it was a homeless man, who then proceeded to mug me. 

0/10 do not recommend

**Comment by m1ndflay3r** [comment]
Update: turns out it was my grandma! 

She really did rob me tho... 0/10

**Comment by m1ndflay3r** [comment]
I ran this code exactly once and immediately it began to break apart reality. I'm now staring at a goddamned Christmas hydra, which this software now speaks though. It has informed me that every vending machine in a 30 km radius is now broken down for literally no reason. I pushed it for answers as to how and why, but it just replied "science gods". 

0/10 non-euclidean ghost code

**Comment by m1ndflay3r** [comment]
Ran this code a few times without too much issue, but then later when I went to McDonald's and ordered a 10 piece McNuggets they only gave me 9 and I didn't notice until I was already home.  3/10 I need that nuggie plzfix

**Comment by m1ndflay3r** [comment]
Rating 0/10 because there was 0 justice for Harambee plsfix

**Comment by m1ndflay3r** [comment]
How do I download more ram??... 2/10

**Comment by m1ndflay3r** [comment]
I ran this EXACTLY ONCE, and it did that do the big to be what you need to get the latest versions of the tiger woods and then I can keep the same way as two. 3/10
